Assets/Plugins/CountlySDK/Services/LocationService.cs

```diff
@@ -141,9 +147,28 @@ public async void SetLocation(string countryCode, string city, string gpsCoordin
             }
         }
 
-        public void DeviceIdChanged(string deviceId, bool merged)
+        /*
+         * If location consent is removed,
+         * the SDK sends a request with an empty "location" parameter.
+         */
+        private async void OnLocationConsentRemoved()

```

Comment:
all location data should be cleared on consent removal

Reply:
ok